code/game/objects/items/weapons/storage/lockbox.dm

```diff
@@ -19,8 +19,8 @@
 	health = 50
 
 /obj/item/weapon/storage/lockbox/proc/damage(var/obj/item/I)
-	if(istype(I, /obj/item/weapon/gun))
-		damaged = 1
+//	if(istype(I, /obj/item/weapon/gun))
+//		I.damaged = 1  //why doesn't this woooork
```

Comment:
Because you haven't cast I to a gun to access the damage var. damage isn't a var on items.